<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostSetupWindow</class>
 <widget class="QDialog" name="HostSetupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="panel" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSpinBox" name="portSpin"/>
      </item>
      <item>
       <widget class="QLabel" name="lblHostInfo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>IP's para comparitlhar</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nickEdit">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Digite seu nome</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="btnPlay">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
